lib/logstash/outputs/sumologic/monitor.rb

```diff
@@ -73,5 +73,9 @@ def build_metric_line(key, value, timestamp)
       "metric=#{key} interval=#{@interval}  category=monitor #{value} #{timestamp}"
     end # def build_metric_line
 
+    def headers()
+      @headers ||= @header_builder.build_stats()
```

Comment:
Ruby... `||=` means that assign if left hand side is `nil`?
Usually this operator would mean "OR left hand side with right hand side and assign it to left hand side of the operator"

Reply:
Yes https://stackoverflow.com/a/14697343/4603021